packages/marketplace/src/utils/__tests__/object.ts

```diff
@@ -0,0 +1,10 @@
+import { cleanObject } from '../object'
```

Comment:
Could we rename this func to something more meaningful. E.g.
`removeKeyWhereValueIsFalsy`

Reply:
I think we should not use a long name for the function. Let me add a function description.